contracts/UniswapUSDCAirdrop.sol

```diff
@@ -0,0 +1,24 @@
+// SPDX-License-Identifier: UNLICENSED
+pragma solidity =0.8.15;
+
+import "./MerkleDistributor.sol";
+import "@openzeppelin/contracts/access/Ownable.sol";
+import "@openzeppelin/contracts/token/ERC20/IERC20.sol";
+
+contract UniswapUSDCAirdrop is MerkleDistributor, Ownable {
```

Comment:
The contract is re-usable by the community for any token and any protocol, so we can call it something more generic

Maybe `MerkleDistributorWithDeadline`?

Reply:
Sure.

Reply:
If it is reusable by any project could the SPDX-Identifier also to the license in the base directory? (GPLv3)